deployment/k8s-config/kustomize/base/skaha/config/catalina.properties

```diff
@@ -6,3 +6,8 @@ ca.nrc.cadc.util.Log4jInit.messageOnly=true
 # (default: ca.nrc.cadc.auth.NoOpIdentityManager)
 ca.nrc.cadc.auth.IdentityManager=ca.nrc.cadc.auth.ACIdentityManager
 
+# uws database connection pool
+#org.opencadc.cavern.uws.maxActive=<maxActive>
+#org.opencadc.cavern.uws.username=<username>
+#org.opencadc.cavern.uws.password=<password>
+#org.opencadc.cavern.uws.url=jdbc:postgresql://ad-uv-01.canfar.net:5432/arcdb
```

Comment:
These shouldn't should be here, they're for cavern only.  But I understand there's an error when not there.  Could you please post the error?  There's probably a config mistake elsewhere.

Reply:
Removed. Didn't see relevant error.

Reply:
Removed